packages/devtools_app/lib/src/code_size/code_size_processor.dart

```diff
@@ -93,3 +76,14 @@ class CodeSizeProcessor {
     return root;
   }
 }
+
+extension DefaultMapGetter<K, V> on Map<K, V> {
+  V lookUpWithDefault(K key, V defaultValue) {
+    final value = this[key];
+    if (_isStringObjectEmpty(value)) return defaultValue;
+    return value;
+  }
+
+  bool _isStringObjectEmpty(V value) =>
+      (value is String) ? value.isEmpty : true;
```

Comment:
this is broken if value is! a string. you want to return false if there is a non-null non-string value.

Reply:
I wasn't sure if we would want to show `defaultValue` or `value` in the case the object is not a string. Anyway, this code is removed now.